src/rnanorm/__main__.py

```diff
@@ -5,6 +5,7 @@
 
 import pandas as pd
 
+from rnanorm.annotation import get_gene_lengths
```

Comment:
Avoid using `get_` prefix. I will add some suggestions below.

Reply:
I was told / convinced that functions should start with verbs. Any reason beside brevity to violate this recommendation?